client/lib/plans/constants.js

```diff
@@ -71,7 +71,7 @@ export const FEATURE_BUSINESS_ONBOARDING = 'business-onboarding';
 export const FEATURE_UPLOAD_PLUGINS = 'upload-plugins';
 export const FEATURE_UPLOAD_THEMES = 'upload-themes';
 export const FEATURE_REPUBLICIZE = 'republicize';
-export const FEATURE_ALL_FREE_FEATURES = 'all-free-features';
+export const FEATURE_ALL_FREE_FEATURES = 'all-free-features'; //PLANS COPY TEST START
```

Comment:
Agree, I don't think it's very helpful outside of this PR's context. That also applies for the rest of the similar inline comments.